Dockerfile

```
@@ -0,0 +1,56 @@
+
+# Set home directory to the app's workdir to ensure writability
+ENV HOME=/app
+
+USER app
+
+# Expose port
+EXPOSE 8000
+
+# Update healthcheck to use the correct endpoint
+HEALTHCHECK --interval=30s --timeout=5s --start-period=10s --retries=3 \
+  [cite_start]CMD python -c "import urllib.request; urllib.request.urlopen('http://localhost:8000/health')" || exit 1 [cite: 3]
```

Hadolint (2.12.0)

[error] unexpected "[ci"
expecting '\', CMD, or at least one space

(DL1000)